### ISSUE TITLE
Add all approved projects to project page

### ISSUE THREAD

**Issue by buckyroberts** [thenewboston-developers/Management#224]
Add all approved projects to the projects page on our website https://thenewboston.com/projects

Also, discuss with @tspearing, @manishram and @Chrisha1212 to collaborate on how we can keep this page updated going forward. The projects are approved by the audit team, then the project needs added to the website, and then we need to have a logo created which is currently done my marketing. After that we need to update the website again with that logo.

After you discuss the ideal flow with the teams and we figure out who is responsible for each step, document the process so that we can share it with any future team members.

**Comment by ma2b0043** [comment]
> Add all approved projects to the projects page on our website https://thenewboston.com/projects
> 

Yeah I was going to do this yesterday but yeah we need a better documented way of doing this. We'll make a workflow document for this and let you know by attaching that doc here.
